src/routes/account/[uid]/+page.server.ts

```diff
@@ -67,8 +67,7 @@ export const actions = {
             ],
             subscription_data: subscriptionData,
             mode: 'subscription',
-            success_url: `${origin}/account/${uid}?subscription_checkout_status=success?session_id={CHECKOUT_SESSION_ID}`,
-            cancel_url: `${origin}/account/${uid}?subscription_checkout_status=cancel`,
+            success_url: `${origin}/blendPro/success?subscription_checkout_status=success?session_id={CHECKOUT_SESSION_ID}`
```

Comment:
idk if we really need these query params with the checkout session ID but left it in for now